WordPress/Jetpack/Resources/release_notes.txt

```diff
@@ -1,6 +1,13 @@
-- We've made it easier to see when there are new blocks available for use on your site
-- You can now highlight text using the <mark> HTML tag, as well as change the text and background color of the Search block button. Shiny!
-- We've hidden HTML tags in post titles when you're checking out your stats.
-- If you're part of a comment thread, you can keep track of it through in-app notifications, not just your email inbox.
-- The Help screen used to break out of its box when rotated from portrait to landscape and back to portrait, but now it's staying put.
-- We fixed more cases where embed blocks would show "unable to display content", allowing you to see all of your content.
+Two-step authentication now comes with extra security. You'll need to unlock your device to approve or deny notifications.
+We added Media File and Attachment Page options to the image block's Link To menu so you can set commonly used URLs.
+We also fixed an issue that made the interface crash when users copied and pasted lists from Microsoft Word.
```

Comment:
I considered moving this added line at the very end, it being a bugfix as opposed to the other items being feature improvements. IMHO it would have put less emphasis on the bugfix if I did so.

But otoh, the wording that was originally provided by Editorial ends with "Finally, we updated the UI […]" so didn't seem right to add an item / line after that "Finally" line…

Any suggestions to find a better balance welcome.

Reply:
```suggestion
We fixed an issue that made the interface crash when users copied and pasted lists from Microsoft Word.
```

Reply:
```suggestion
We fixed an issue that made the interface crash when users copied and pasted lists from Microsoft Word.
```

To align with what's in the `.po` file?

Reply:
Nice catch! I indeed messed up between various copy/pastes, will remove the `also`.